test/run-builder.sh

```diff
@@ -22,8 +22,22 @@ else
 fi
 
 builder_start() {
-  GATEWAY_IP=$(${CONTAINER_RUNTIME} network inspect org.osbuild.koji | jq -r ".[0].plugins[0].ipam.ranges[0][0].gateway")
+  set +e
+  # Podman v4 introduced breaking changes to network option
```

Comment:
If we switched the CI to fedora 36 (which is what Fedora rel-eng is switching to slowly, we wouldn't probably need all of this, would that be possible?

Reply:
Yeah I thought about this option as well, but we don't have F-36 runners yet in `gitlab-ci-terraform` I'll add them then and ask for review there :)

Reply:
Adding runners in https://github.com/osbuild/gitlab-ci-terraform/pull/66